message/router/middleware/recoverer.go

```diff
@@ -26,7 +25,7 @@ func Recoverer(h message.HandlerFunc) message.HandlerFunc {
 
 		defer func() {
 			if r := recover(); r != nil || panicked {
-				err = errors.WithStack(RecoveredPanicError{V: r, Stacktrace: string(debug.Stack())})
```

Comment:
This might be a bit of a breaking change, actually. 🤔 Perhaps we could somehow keep the stack trace here?

Reply:
I'm not sure how this really is a breaking change or not.
A `debug.Stack` call is used to get the stacktrace into `RecoveredPanicError`. (via a call to [`runtime.Callers`](https://pkg.go.dev/runtime#Callers)).
This seems to me like the same function that [`errors.WithStack`](https://github.com/pkg/errors/blob/master/errors.go#L145) provides as it's using `runtime.Callers` and then formatting it.

Now, should someone use reflection or have an interface to represent and error from the errors package then yes this PR could break things. 
Personally, I feel this change is not breaking the contract of the library as it's still returning the `error` interface.